invenio_communities/records/api.py

```diff
@@ -245,13 +245,16 @@ def as_dict(self, include_requests=True):
         res = {
             **self,
             'status': str(self.status.title),
-            'record_pid': self.record.pid.pid_value,
-            'community_pid': self.community.pid.pid_value
+            'recid': self.record.pid.pid_value,
+            'comid': self.community.pid.pid_value,
+            'community_title': self.community['title'],
+            'record_title': self.record['titles'][0]['title'],
```

Comment:
This is specific to each record's metadata schema, which we cannot know in advance (i.e. we can put new things in the record, but cannot assume how other fields of the record look like). Also, this doesn't seem to be used anywhere.